### ISSUE TITLE
const updateNotifier = require('update-notifier');

### ISSUE THREAD

**Issue by jJit0** [devXprite/infoooze#20]
Hi,

When updated infoooze to 1.2.0 I have this error:
```
/usr/local/lib/node_modules/infoooze/src/home.js:4
const updateNotifier = require('update-notifier');
                       ^

Error [ERR_REQUIRE_ESM]: require() of ES Module /usr/local/lib/node_modules/infoooze/node_modules/update-notifier/index.js from /usr/local/lib/node_modules/infoooze/src/home.js not supported.
Instead change the require of index.js in /usr/local/lib/node_modules/infoooze/src/home.js to a dynamic import() which is available in all CommonJS modules.
    at Object.<anonymous> (/usr/local/lib/node_modules/infoooze/src/home.js:4:24)
    at Object.<anonymous> (/usr/local/lib/node_modules/infoooze/index.js:5:26) {
  code: 'ERR_REQUIRE_ESM'
}
```

Any ideas ?

**Comment by devXprite** [comment]
Thanks @jJit0 for reporting this bug.
Now this bug had been fixed in [v1.2.1](https://www.npmjs.com/package/infoooze/v/1.2.1)

**Comment by jJit0** [comment]
All good for me, it works now :smile:
